Mittons.Fixtures.Tests.Unit/Docker/Containers/BaseSftpContainerTests.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Mittons.Fixtures.FrameworkExtensions.Xunit.Docker.Containers;
+
+namespace Mittons.Fixtures.Tests.Unit.Docker.Containers;
+
+public abstract class BaseSftpContainerTests : IAsyncDisposable
```

Comment:
Yeah, that came about because of the  Xunit specific SftpContainer not extending the Xunit specific Container.
I have another way to change that but I'm hoping to get a more across-the-board solution for this soon.